tools/scripts/checkSurrogateExists.py

```diff
@@ -0,0 +1,36 @@
+#!/bin/python3
+import os
+import sys
+
+srcdir = "../../src/catch2"
+surrogates_folder = "../../tests/SurrogateCpps"
+header_files = []
+surrogate_files = []
+throw_error = False
+
+def get_files(filetype,dir):
+    filenames = []
+
+    for _, _, files in os.walk(dir):
+        for file in files:
+            if filetype in file:
+                filenames.append(file)
+
+    return filenames
```

Comment:
Is there an advantage to this approach compared to using [`glob`](https://docs.python.org/3/library/glob.html)?

Reply:
I don't know how glob works, but I believe there will be no difference for the number of files in Catch2's case.